config/premierp550.toml

```diff
@@ -0,0 +1,26 @@
+# A simple configuration to run on the premier P550 board
+
+[log]
+level = "info"
+color = true
+
+[debug]
+# max_firmware_exits = 2000
+
+[vcpu]
+max_pmp = 0
+
+[platform]
+name = "premierp550"
+nb_harts = 5
```

Comment:
Do we know if we have 4 or 5 cores? We can change that later, but it would be interesting to know. The VisionFive has 5, but one is smaller than the main 4, I am not sure if the P550 is similar.